tox.ini

```diff
@@ -159,3 +158,12 @@ commands =
 whitelist_externals =
     rm
     {[testenv]whitelist_externals}
+
+[testenv:changelog]
+skip_install = false
+usedevelop = false
+description = Generate the changelog
+extras =
+  changelog
```

Comment:
I believe that for towncrier to work installing dist is not required. So, if you remove that, this task will complete faster.

Currently, there's no way to tell pip "install extras but not molecule itself" which means that it should probably be left out of dist metadata.